### ISSUE TITLE
multi analyzers in analyzers.yml cause error

### ISSUE THREAD

**Issue by LuckyC4t** [praetorian-inc/gokart#58]
first i add some analyzers in yml
![image](https://user-images.githubusercontent.com/40694340/139442177-6bc8c455-dca9-46db-b06f-9d552033b533.png)
then, i add some debug code in generic.go
```go
func genericFunctionRun(pass *analysis.Pass, vulnPathFuncs map[string][]string,
	name string, message string) (interface{}, error) {
	fmt.Println(name, message) <- add debug code
        ...
```
run gokart to scan a test project, and this project only has two files: go.mod, main.go
```bash
gokart scan -i /foo/bar/analyzers.yml /foo/testproject/
```
expect output:
```
Revving engines VRMMM VRMMM
3...2...1...Go!
Test Sink1 test1
Test Sink2 test2
```
actual output:
```
Revving engines VRMMM VRMMM
3...2...1...Go!
Test Sink2 test2
Test Sink2 test2
```

**Comment by isp1r0** [comment]
@LuckyC4t can you confirm that this issue is fixed with your PR #59?  I wasn't positive whether the closure fix also fixed this issue.  Reopening until confirmation... Thanks!

**Comment by LuckyC4t** [comment]
hi! i pull the latest project, and use goland to debug it. it looks correct.

![image](https://user-images.githubusercontent.com/40694340/141397781-63d7700e-3dbe-4eaf-9bf9-6c222ba1fca2.png)
![image](https://user-images.githubusercontent.com/40694340/141397800-534e07c7-a884-4d79-af53-97dc5c638dc2.png)
